File: VSTemplate/VSExtension/ItemTemplates/SOLIDWORKSPDMTaskPageVBNET/TaskPageViewModel.vb
Imports System.ComponentModel
Imports System.Runtime.CompilerServices

Namespace $rootnamespace$
    Public Class $safeitemname$
        Implements INotifyPropertyChanged

        Public Event PropertyChanged As PropertyChangedEventHandler Implements INotifyPropertyChanged.PropertyChanged

        Public Sub OnPropertyChanged(<CallerMemberName> Optional propertyName As String = "")
            RaiseEvent PropertyChanged(Me, New PropertyChangedEventArgs(propertyName))
        End Sub

    End Class
End Namespace

File: VSTemplate/VSExtension/ItemTemplates/SOLIDWORKSPDMTaskPage/TaskPageViewModel.cs
using System.ComponentModel;
using System.Runtime.CompilerServices;

namespace $rootnamespace$
{
    public class $safeitemname$ : INotifyPropertyChanged
    {
        public event PropertyChangedEventHandler PropertyChanged;

        public void OnPropertyChanged([CallerMemberName] string propertyName = "")
        {
            PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
        }
    }
}
